src/axes/timeAxis.ts

```diff
@@ -483,7 +482,7 @@ export module Axes {
       this._renderLabellessTickMarks(labelLessTicks);
 
       this._hideOverflowingTiers();
-      for (i = 0; i < tierConfigs.length; ++i) {
```

Comment:
A modification was needed here.

The reason was because `i` was defined in the above for loop, which means that this `i` has lost scope of the previous scope after moving to `let`